index.html

```diff
@@ -1209,6 +1115,29 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
                 while other apps will see statements from public and also
                 private extended profile documents they have access to.
               </p>
+              <p>
+                Profiles can contain, and apps are free to follow, any kind of
+                links to related documents. In order to promote interoperability
+                and limit the burden on apps, this specification recommends a
+                limited number of related documents which a profile ought to
+                contain so that apps can discover them.
+              </p>
+              <p>
+                The discovery process starts with the WebID, a URI that points
+                to exactly one document, referred to here as a WebID Profile
+                Document. This document can be expected to contain pointers to a
```

Comment:
```suggestion
                The discovery process starts with the WebID, a URI that points
                to exactly one document, referred to here as a WebID Profile
                Document [<cite><a class="bibref" href="#bib-webid">WEBID</a></cite>]. This document can be expected to contain pointers to a
```

And update similarly if need to be clear.